models/stg_twitter_ads.yml

```diff
@@ -152,7 +190,13 @@ models:
       - name: is_latest_version
         description: Whether the record is the most latest version for the promoted tweet.
   - name: stg_twitter_ads__promoted_tweet_report
-    description: Each record represents the performance of a promoted tweet on a given day.
+    description: Each record represents the performance of a promoted tweet on a given day, in the tweet's defined placement.
```

Comment:
It looks like `_fivetran_synced` is documented, but it isn't actually used in the staging model. Can we either remove it from the docs or add it to the staging model. 
![image](https://user-images.githubusercontent.com/74217849/181789270-bd3482f6-5f65-4603-9010-dfb4ed415978.png)

Reply:
removed